src/lib/locals.js

```diff
@@ -8,9 +8,16 @@ const {
   GA4_DISABLED,
   DT_RUM_URL,
   LOGOUT_URL,
+  DELETE_ACCOUNT_URL,
 } = require("./config");
 const { generateNonce } = require("./strings");
 
+const generateUrlFromString = (stringURl, req) => {
```

Comment:
This method should be `addFromUrlParam` or similar, I think?

Do we know whether that's desirable for the delete account link - it feels like it might be specific to the contact form...